docs/rst/recording/usage/configuration.rst

```diff
@@ -370,6 +370,10 @@ Enabling ``file-rotation`` allows the |ddsrecorder| to overwrite old files to fr
 
     If the *DDS Recorder's* state changes to ``stop`` while ``file-rotation`` is set, the |ddsrecorder| will save its current output files from being overwritten in the future.
 
+.. note::
+
+    If an output file is moved, deleted, or renamed, the |ddsrecorder| will account for the size of the file in question and rotate between the remaining files.
```

Comment:
"will account for" doesn't say much to me. It is true I'm not a native speaker though.